src/components/ArticleCard.jsx

```diff
@@ -35,6 +39,38 @@ export default function ArticleCard() {
     );
   }
 
+  function handleUpVote() {
+    const newVote = voted + 1;
+    patchVotesByArticleId(article_id, 1)
+      .then(() => {
+        setVoted(newVote);
+        setArticle((currArticle) => {
+          return { ...currArticle, votes: currArticle.votes + 1 };
+        });
+        setVoteMessage("Upvoted! Nice!");
+      })
+      .catch((err) => {
+        console.log(err);
+        setVoteMessage("Oooops! Something went wrong. Try again later.");
```

Comment:
These error messages dont seem to be coming through? If we use inspect and then set chrome toe offline mode they dont show, will drop in to explain this.